share/packages/qmpackage_defaults.xml

```diff
@@ -1,34 +1,40 @@
-<package>
-<name>xtp</name>
-<charge>0</charge>
-<spin>1</spin>
+<package help="Defaults for the Quantum Mechanics packages">
+<name help="QM package to use" default="xtp" choices="xtp,orca">xtp</name>
+<charge help="Molecular charge" default="0" choices="int">0</charge>
+<spin help="Molecular multiplicity" default="1" choices="int+">1</spin>
 <basisset help="Basis set for MOs" default="def2-tzvp">def2-tzvp</basisset>
 <auxbasisset help="Auxiliary basis set for RI" default="aux-def2-tzvp">aux-def2-tzvp</auxbasisset>
-<optimize>false</optimize>
+<optimize help="Perform a molecular geometry optimization" default="false" choices="bool">false</optimize>
 <functional default="XC_HYB_GGA_XC_PBEH">XC_HYB_GGA_XC_PBEH</functional>
 <scratch help="path to the scratch folder" default="/tmp/qmpackage">/tmp/qmpackage</scratch>
 <polarisation>false</polarisation>
```

Comment:
Why are `polarisation` and `dipole_spacing` in `qmpackage`. 
(Also, can we stick to American English?)

Reply:
So the polarisation bool had a purpose, but I think it should be removed and we should just check if a induced dipole exists and then split it up. So that bool should go because it is unintuitive.

The reason why `dipole_spacing` is in qmpackage is xtpdft does not need it, so it is to some extend a package specific option. `dipole_spacing` essentially describes how a qmpackage should handle higher multipoles.

Reply:
I am going to change the `options` to American English